README.md

```diff
@@ -32,15 +32,19 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? 
+I would say it makes it readable for other developers and organizes your html so its not a jumbled mess.
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+Block encloses content into block sections where inline doesnt it runs it continusly in a line with no line breaks.
```

Comment:
More specifically:

* Block elements take up the whole with of the container they're in. That means any elements would be forced to be under it, creating a column.
* Inline elements take up only the amount of width needed to fit its content. That means it is _possible_ to place other inline elements beside it to create a row.